src/modules/components/proposal/proposalActions/actions/proposalActionTokenMint/proposalActionTokenMint.test.tsx

```diff
@@ -0,0 +1,118 @@
+import { render, screen } from '@testing-library/react';
+import { modulesCopy } from '../../../../../assets';
+import { OdsModulesProvider } from '../../../../odsModulesProvider';
+import { type IProposalActionTokenMint } from '../../proposalActionsTypes';
+import { generateProposalActionTokenMint } from '../generators/proposalActionTokenMint';
+import { ProposalActionTokenMint, type IProposalActionTokenMintProps } from './proposalActionTokenMint';
+
+jest.mock('../../../../member/memberDataListItem/memberDataListItemStructure', () => ({
+    MemberDataListItemStructure: () => <div data-testid="member-data-list-item" />,
+}));
+
+describe('<ProposalActionTokenMint /> component', () => {
+    const createTestComponent = (props?: Partial<IProposalActionTokenMint>) => {
```

Comment:
The component should accept the props instead of the action so that we can customise the component if we add other props later on
```suggestion
    const createTestComponent = (props?: Partial<IProposalActionTokenMintProps>) => {
```